start-fuzzing/action.yml

```diff
@@ -19,6 +19,10 @@ inputs:
   fuzzing_artifact:
     description: "Fuzzing artifact path. Used when building the fuzzing artifact in a github workflow."
     required: false
+  checkout_directory:
+    description: "Directory of the repository checkout. Relative to the github workspace."
+    required: false
+    default: ""
```

Comment:
I assume that by default, the GitHub workspace is used as the checkout directory. If that's correct, we should mention that in the description IMO.